### PR TITLE
dev-db/influxdb: Version bump to 1.8.10

### PR DESCRIPTION
**Pull request by Dwosky**
Basically changed `EGO_SUM` for deps package like the [go-module](https://devmanual.gentoo.org/eclass-reference/go-module.eclass/index.html) documentation suggests, while also bumping the package to last released minor version.

Currently deps package its hosted in my GitHub Pages repository, so it should also be copied to some mirror and update the ebuild acordingly.

Also I checked all the dependency licences with [golicense](https://github.com/mitchellh/golicense) and the only change I saw was `EPL-1.0` of previous ebuild to `EPL-2.0` as sugested by the report. If someone has some spare time, it would be great to check this in order to verify its all ok.

Closes: https://bugs.gentoo.org/837131
Signed-off-by: Pedro Arizmendi <dwosky@pm.me>
Package-Manager: Portage-3.0.30, Repoman-3.0.3

### CODE REVIEW

File: dev-db/influxdb/influxdb-1.8.10.ebuild [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	app-text/xmlto
+)"
+COMMON_DEPEND="
+	acct-group/influxdb
+	acct-user/influxdb"
+DEPEND="${COMMON_DEPEND}"
+RDEPEND="${COMMON_DEPEND}"
+
+src_compile() {
+	GOBIN="${S}/bin" \
```
Comment (juippis) [inline dev-db/influxdb/influxdb-1.8.10.ebuild:31]: Uhh, what's up with this?

File: dev-db/influxdb/influxdb-1.8.10.ebuild [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit go-module systemd
```
Comment (juippis) [inline dev-db/influxdb/influxdb-1.8.10.ebuild:5]:
```suggestion
EAPI=8

inherit go-module systemd
```

File: dev-db/influxdb/influxdb-1.8.10.ebuild (hunk 1 above)
Comment (Dwosky) [reply dev-db/influxdb/influxdb-1.8.10.ebuild:31]: Sorry, mind explaining the issue? Is the dependency (which doesn´t look great right now) or its the way `src_compile` its configured?

I've reused the old ebuild, so there might be items to correct/enhance that I wasn't aware of.

Comment (juippis) [reply dev-db/influxdb/influxdb-1.8.10.ebuild:31]: It's the `GOBIN="${S}/bin"` part. I'm not super familiar with how go works in Gentoo, but it **looks** like this uses some bundled binary to compile the end-product. Which looks suspicious to be honest. Although I'm not familiar whether `GOBIN` does anything at all with *go-module.eclass*, or whether it's some go-internal thing.

Comment (Dwosky) [reply dev-db/influxdb/influxdb-1.8.10.ebuild:31]: According to the documentation, `GOBIN` variable defines the directory where `go install` and `go get` will place binaries after building `main`. I suppose its defined so line 41 gets the binary to install from there.

William Hubs is both maintainer of this package as well as author/maintainer of the _go-module.eclass_, that's why I didn't messed much with the ebuild in the first place.

Comment (juippis) [reply dev-db/influxdb/influxdb-1.8.10.ebuild:31]: Okay, thanks for explaining. Makes more sense, even though it seems a bit redundant imho (This should be handled by a function outside ebuild).

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**